# distance in mm between the gripper points
float64 distance
float64 maxMmPerSecond